packages/component_code_gen/config/config.py

```diff
@@ -1,21 +1,48 @@
+import os
 from dotenv import load_dotenv
 load_dotenv()
 
-import os
+
+DEFAULTS = {
+    "OPENAI_API_TYPE": "openai",
+    "OPENAI_MODEL": "gpt-4",
```

Comment:
Can this be set from `.env`? And does this model string specifically conflict with the 32k model declaration there?